app/kumactl/cmd/get/get_dataplane.go

```diff
@@ -0,0 +1,52 @@
+package get
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Kong/kuma/app/kumactl/pkg/output"
+	"github.com/Kong/kuma/app/kumactl/pkg/output/printers"
+	rest_types "github.com/Kong/kuma/pkg/core/resources/model/rest"
+	"github.com/Kong/kuma/pkg/core/resources/store"
+)
+
+func newGetDataplaneCmd(pctx *getContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dataplane NAME",
+		Short: "Show Single Dataplane Resource",
```

Comment:
how about `Show a single Dataplane resource`. No need for uppercase Single and Resource. Can you make this change in every command? Thank you.